doc/markdown/commandline.md

```diff
@@ -58,6 +58,7 @@ All the options can also be set with code (defaults/overrides) if the user [**su
 | ```-ns``` &nbsp; ```--no-skip=<bool>``` | Don't skip test cases marked as skip with a decorator |
 | ```-gfl``` ```--gnu-file-line=<bool>``` | ```:n:``` vs ```(n):``` for line numbers in output (gnu mode is usually for linux tools/IDEs and is with the ```:``` separator) |
 | ```-npf``` ```--no-path-filenames=<bool>``` | Paths are removed from the output when a filename is printed - useful if you want the same output from the testing framework on different environments |
+| ```-spp``` ```--skip-path-prefixes=<bool>``` | Remove the longest matching one of these prefixes from any file path in the output - useful as ```-npf```, but can preserve some context through relative paths. Try: ```--spp=${CMAKE_SOURCE_DIR}/:${CMAKE_BINARY_DIR}/``` |
```

Comment:
this should not be a <bool> but a <string>

And perhaps should be changed from ```useful as -npf``` to ```similar to -npf``` and from ```but can preserve some context through relative paths``` to ```but can preserve some context by not removing the entire relative paths``` or something like that

Reply:
I'll also add a hint at the `:` separator.